docs/src/main/paradox/release-notes.md

```diff
@@ -0,0 +1 @@
+# Release Notes
```

Comment:
we will not have release notes in docs here, remove file

Reply:
removed